paymentapi/src/main/java/fi/hel/verkkokauppa/payment/api/PaymentAdminController.java

```diff
@@ -166,4 +171,21 @@ public ResponseEntity<VismaPayResponse> cancelPayment(@RequestParam(value = "pay
         }
     }
 
+    @PostMapping(value = "/payment-admin/online/save-payment-filter", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<PaymentFilterDto>> savePaymentFilter(@RequestBody @Valid List<PaymentFilterDto> paymentFilters) {
+        try {
+            List<PaymentFilter> model = filterService.savePaymentFilters(paymentFilters);
+            List<PaymentFilterDto> dto = filterService.mapToDto(model);
+            return ResponseEntity.status(HttpStatus.CREATED).body(dto);
```

Comment:
But then we just say null is created and that logic is not good in my opinion?

Think this as developer, you send api request and get 201 status but response is null

Reply:
Well, a list can never be null in Java (singular entries on the other hand can), so it would return empty list if an empty array would be sent to the endpoint or the list would be defined but not filled. However, I do agree that sending in an empty list and getting a 201 and an empty list is not logical, so a check could be implemented to return a more sensible code and message along the lines "You need to specify at least one payment filter."